boards/posix/nrf52_bsim/Kconfig

```diff
@@ -0,0 +1,32 @@
+
+if BOARD_NRF52_BSIM
+
+comment "NRF52_BSIM options"
+
+config PRINTK_HOOK_INIT_PRIORITY
+	int
+	depends on BOARD_NRF52_BSIM
+	default 50
+	help
+	  Just the driver init priority
+
+endif # BOARD_NRF52_BSIM
+
+
+#This would eventually be shared by a possible family of simulated NRF boards
+#which use BabbleSim. When that happens, we can move this to a common
+#Kconfig file
```

Comment:
Nit: Putting a space after `#` would make it consistent with the other comments.